### PR TITLE
config fix

### PR DESCRIPTION
**Pull request by kellerza**
Bugfixes

- TemplatePaths inject @ earlier, enable GetTemplateNameinDirs
- "role" --> "clab_role"
- "links" --> "clab_links"
- ipFarendS - fix invalid IPPrefix

### CODE REVIEW

File: clab/config/utils.go [hunk 1]
```diff
@@ -122,6 +122,7 @@ func prepareLinkVars(lIdx int, link *types.Link, varsA, varsB Dict) error {
 			// Calculate the remote IP
 			vs := fmt.Sprintf("%v", v)
 			ipF, err := ipFarEndS(vs)
+			fmt.Printf("%s -- %s\n", vs, ipF)
```
Comment (hellt) [inline clab/config/utils.go:125]: leftover debug statement?

File: clab/config/utils.go [hunk 2]
```diff
@@ -257,12 +262,14 @@ func ipFarEnd(in netaddr.IPPrefix) netaddr.IPPrefix {
 // template names are following the pattern <some-name>__<role/kind>.tmpl
 func GetTemplateNamesInDirs(paths []string) ([]string, error) {
 	var tnames []string
+	fmt.Printf("%v", paths)
```
Comment (hellt) [inline clab/config/utils.go:265]: leftover debug?

File: clab/config/utils.go [hunk 3]
```diff
@@ -257,12 +262,14 @@ func ipFarEnd(in netaddr.IPPrefix) netaddr.IPPrefix {
...
 	for _, p := range paths {
 		all, err := filepath.Glob(filepath.Join(p, "*__*.tmpl"))
 		if err != nil {
 			return nil, err
 		}
 		for _, fn := range all {
+			fmt.Printf("%s", fn)
```
Comment (hellt) [inline clab/config/utils.go:272]: leftover debug?

**Comment by hellt** [comment]
it good with me,
let me know if it is ready to be merged @kellerza

**Comment by kellerza** [comment]
Should be good to go, thx

**Pull request merged**